lib/livebook/config.ex

```diff
@@ -276,7 +274,7 @@ defmodule Livebook.Config do
   @spec logout_enabled?() :: boolean()
   def logout_enabled?() do
     {_type, module, _key} = Livebook.Config.identity_provider()
-    authentication().mode != :disabled or module.logout_supported?()
+    Code.ensure_loaded?(module) and function_exported?(module, :logout, 2)
```

Comment:
We still want to check both!

```suggestion
    identity_logout? =
      Code.ensure_loaded?(module) and function_exported?(module, :logout, 2)

    authentication().mode != :disabled or identity_logout?
```